pkg/controller/elasticsearch/nodespec/defaults.go

```diff
@@ -40,12 +40,14 @@ var (
 			corev1.ResourceMemory: DefaultMemoryLimits,
 		},
 	}
+)
 
-	// DefaultAnnotations are the default annotations for the Elasticsearch pods
-	DefaultAnnotations = map[string]string{
+// defaultAnnotations are the default annotations for the Elasticsearch pods
+func defaultAnnotations() map[string]string {
```

Comment:
I've been tempted to inline it since it is only used at a single place.

Reply:
I think that would be reasonable.